content/en/docs/ambient/usage/waypoint/index.md

```diff
@@ -9,4 +9,339 @@ owner: istio/wg-networking-maintainers
+simply add the `--enroll-namespace` parameter to your `istioctl experimental waypoint apply` command, which labels the namespace with `istio.io/use-waypoint: waypoint` for you automatically:
+
+{{< text bash >}}
+$ istioctl experimental waypoint apply -n default --enroll-namespace
+waypoint default/waypoint applied
+namespace default labeled with "istio.io/use-waypoint: waypoint"
+{{< /text >}}
+
+Or you can add the `istio.io/use-waypoint: waypoint` label to the `default` namespace using `kubectl`:
+
+{{< text bash >}}
+$ kubectl label ns default istio.io/use-waypoint=waypoint
+namespace/default labeled
+{{< /text >}}
+
+After the namespace is enabled for waypoint, the waypoint proxy can be used for L7 processing for any services running in the namespace. For any requests from any pods in ambient to any service in the `default` namespace, the requests must go through the `waypoint` for L7 processing and policy enforcement.
+
+If you prefer more granularity than namespace waypoint, you can label your specific service or pod in the namespace to use a different waypoint. For example, you may want your `WasmPlugin` resource to apply only on a specific service or you are calling a Kubernetes
+[headless service](https://kubernetes.io/docs/concepts/services-networking/service/#headless-services) by its pod IP address.
+
+### Configure a specific service with its own waypoint
+
+Deploy a waypoint called `reviews-svc-waypoint` for the `reviews` service:
+
+{{< text bash >}}
+$ istioctl experimental waypoint apply -n default --name reviews-svc-waypoint
+waypoint default/reviews-svc-waypoint applied
+{{< /text >}}
+
+Label the `reviews` service to use the `reviews-svc-waypoint` waypoint:
+
+{{< text bash >}}
+$ kubectl label service reviews istio.io/use-waypoint=reviews-svc-waypoint
+service/reviews labeled
+{{< /text >}}
+
+For any requests from any pods in ambient to the `reviews` service, the requests must go through the `reviews-svc-waypoint` for L7 processing and policy enforcement.
+
+### Configure a specific pod with its own waypoint
+
+Deploy a waypoint called `reviews-v2-pod-waypoint` for the `reviews-v2` pod:
+
+{{< text bash >}}
+$ istioctl experimental waypoint apply -n default --name reviews-v2-pod-waypoint --for workload
+waypoint default/reviews-v2-pod-waypoint applied
+{{< /text >}}
+
+Label the `reviews-v2` pod to use the `reviews-v2-pod-waypoint` waypoint:
+
+{{< text bash >}}
+$ kubectl label pod -l version=v2,app=reviews istio.io/use-waypoint=reviews-v2-pod-waypoint
+pod/reviews-v2-5b667bcbf8-spnnh labeled
+{{< /text >}}
+
+For any requests from any pods in ambient to the `reviews-v2` pod IP, the requests must go through the `reviews-v2-pod-waypoint`
+for L7 processing and policy enforcement.
+
+## Attach L7 policies to waypoint proxies {#attachl7policies}
+
+The following L7 policies are supported for waypoint proxy:
```

Comment:
We normally use "policies" to refer to authn/authz etc; the top 3 here are "request routing".  The following L7 features?

(And with links back to the docs in which they are defined, though I don't think we have those)

Reply:
I do think http routing policies are L7 policies. Not sure about TCP or TLSRoute but today they require a waypoint so this table is correct to my knowledge.

Reply:
Ahh, this is a new Gateway API thing they didn't consult me on!